### PR TITLE
Join us - Volunteer Expectations

### PR DESCRIPTION
**Pull request by kimrass14**
https://trello.com/c/37w1qzyc

I reused the classes from the Work With Us icon section to maintain consistent styling, per the card. To avoid confusion, I renamed some of the Work With Us classes so they are more generic and could be used in the Work With Us and Join Us sections.

### CODE REVIEW

File: pages/join.html [hunk 1]
```diff
@@ -5,46 +5,29 @@
 home: false
 ---
 
-<h2>Join content goes here.</h2>
-
-<div class="contentContainer">
-  <div class="cardsContainer">
```
Comment (daleinen7) [inline pages/join.html:11]: It looks like this section was from a previous [ticket](https://trello.com/c/VuBaiJvq/113-join-us-project-cadence) and I think it goes above the section this PR is for. We should keep it in.

File: pages/join.html [hunk 2]
```diff
@@ -5,46 +5,29 @@
...
-            <h4 class="cardTitle">LOREM IPSUM</h4>
-            <h2 class="cardText">Who leads our teams?</h2>
-            <p class="cardText">Project and Product Managers lead our teams. They help define the roadmap and create tickets in our kanban board. We'll try to define MVPs and create tasks to get there.</p>
-          </div>
-        </div>
-        <div class="col-md-auto card-img-container">
-          <img src="/images/proxy-img.png" alt=""/>
-        </div>
+<div class="volunteer-expectations">
+  <h2 class="volunteer-title">What are volunteer <br>expectations?</h2>
```
Comment (daleinen7) [inline pages/join.html:9]: I think this styles fine without the break tag. We should leave it out.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Nice job making those classes more reusable. Just saw a few things to fix. Otherwise looks good to me.

File: pages/join.html (hunk 2 above)
Comment (kimrass14) [reply pages/join.html:9]: removed <br>

File: pages/join.html [hunk 3]
```diff
@@ -5,46 +5,29 @@
 home: false
 ---
 
-<h2>Join content goes here.</h2>
-
-<div class="contentContainer">
-  <div class="cardsContainer">
-    <div class="padded-content cards">
```
Comment (daleinen7) [inline pages/join.html:12]: One more thing. It looks like this section was from a [previous ticket](https://trello.com/c/VuBaiJvq/113-join-us-project-cadence) that goes above the section this PR references.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
One more part. I think there is some stuff that is relevant still to the join page.

File: pages/join.html (hunk 3 above)
Comment (kimrass14) [reply pages/join.html:12]: I added .contentContainer as a parent div. I left .cardsContainer and .cards out bc it added duplicative padding. Padded-content is included.

Comment (daleinen7) [reply pages/join.html:12]: It still looks like the "project cadence" section is removed in this branch. 
Screen Shot 2021-05-25 at 3 50 42 PM

Comment (kimrass14) [reply pages/join.html]: Ok, I think I finally got it...added back project cadence

**Review by daleinen7 — APPROVED** [review]
Looks good to me!

**Pull request merged**